src/zero/module/copyfile.clj

```diff
@@ -47,5 +47,5 @@
                 (jdbc/update! tx items
                               {:updated now :uuid uuid}
                               ["id = ?" id])))]
-      (let [workflow (postgres/get-table tx items)]
+      (if-let [workflow (postgres/get-table tx items)]
```

Comment:
I feel that the if-expression requires some handling for the case when workflow is `nil`. Is there a situation when `nil` is valid at this point? If not, should we just let it fail rather than program defensively? Can we handle this case at this point rather than fail silently?

Reply:
Note this also applies to the other call sites where there's an `if-let` now :)

Reply:
That's a good point! Since the workload "items" table gets created when the workload is created, this call to `get-table` should not return `nil` unless something is wrong with the database